dev/validate_bom__config.py

```diff
@@ -45,6 +45,7 @@
 
 import logging
 import os
+import textwrap
```

Comment:
Can remove this import.

Reply:
I mentally removed it; I'll remove it from the actual code now